t/app/controller/waste_kands_bulky.t

```diff
@@ -418,6 +419,8 @@ FixMyStreet::override_config {
             is $new_report->get_extra_metadata('scpReference'), '12345', 'correct scp reference on report';
 
             $mech->clear_emails_ok;
+            FixMyStreet::Script::Alerts::send_updates();
+            is $mech->email_count_is(0), 1, 'Update cancelled as would go to body user not report user';
```

Comment:
The update /would/ go to the report user, it's cancelled because we don't want to send it to them in this case.

Reply:
Actually this test is unaffected by whether the update is cancelled or not - when I implemented I was pretty sure it failed when I didn't have the cancel_alert implemented. Not sure where I've got confused

Reply:
You're running this too early - the update isn't created until the call to pay_complete a few further lines down.
And then you haven't actually changed which user is creating the comment to the body user, so the alert wouldn't be sent as is same user as problem.